src/main/java/baritone/BaritoneProvider.java

```diff
@@ -59,7 +60,7 @@ public List<IBaritone> getAllBaritones() {
 
     @Override
     public IWorldScanner getWorldScanner() {
-        return WorldScanner.INSTANCE;
+        return FasterWorldScanner.INSTANCE;
```

Comment:
I can't seem to find any usages of the old WorldScanner, so it doesn't really make sense to me why your calling this FasterWorldScanner, surely it should just replace the old world scanner.

Reply:
yeah. really it's a temporary name so it's easier to directly benchmark and compare the differences